# Define the goal
int8 target_ID
float32[] pos
int64 time
--- 
# Define the result
bool success
---
# Define a feedback message
float32 percent_complete